ethereum/gnosis_protocol_v2/trades.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE gnosis_protocol_v2.trades
+(
+    app_data           text,
+    atoms_bought       integer     NOT NULL,
+    atoms_sold         integer     NOT NULL,
```

Comment:
Did a testrun with the `insert` function and got the following error:
>  ERROR:  integer out of range

Unfortunately, it is not clear from the error which of the `integer` fields in the table has the `out of range` problem.

Background info: [html](https://www.postgresql.org/docs/13/datatype-numeric.html)

Reply:
I think in need to use numeric for these large wei values.